package/etc/conf.d/conflib/_common/templates.conf

```diff
@@ -34,6 +34,10 @@ template t_everything {
         template("${ISODATE} ${HOST} ${LEGACY_MSGHDR}${MESSAGE}");
         };
 
+template t_cef_hdr_msg_for_esa {
```

Comment:
this isn't needed to get the original layout simply use t_legacy_hdr_msg

Reply:
The CEF header is not getting ingested using **t_legacy_hdr_msg** template. Hence we have created **t_cef_hdr_msg_for_esa** template.